packages/errors/src/is-directus-error.ts

```diff
@@ -1,20 +1,26 @@
 import type { DirectusError } from './create-error.js';
+import type { ExtensionsMap } from './types.js';
 
 /**
- * Check whether or not a passed thing is a valid Directus error
+ * Check whether or not a passed value is a valid Directus error.
  *
- * @param err - Any input
+ * @param value - Any value
+ * @param code - Error code to check for
  */
-export const isDirectusError = <T = unknown>(err: unknown, code?: string): err is DirectusError<T> => {
+
+export const isDirectusError = <T = never, C extends string = string>(
```

Comment:
Should we convert this to `function isDirectusError` since that seems to be the way we implement new functions?

Reply:
Good question 😅 I think this is a bit of an unsettled matter. For me, both works 👍

Reply:
Same!

Reply:
Great, then I think we can keep it as is 🚀